ngraph/core/include/ngraph/op/util/variable.hpp

```diff
@@ -14,6 +14,12 @@ namespace ngraph
         PartialShape data_shape;
         element::Type data_type;
         std::string variable_id;
+
+        inline bool operator==(const VariableInfo& other) const
```

Comment:
Could we add nGraph unit tests for this operator?

Reply:
Added.